index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header>
+            <a class="logo" href="#"><p>My logo</p></a>
+            <img src="./img/Icon - Menu.svg" alt="mobile menu">
+    </header>
+    <section class="headline">
+        <div class="corps"></div>
+        <h1>I’m Mehdi jdir. <br>
+            Glad to see you!
+        </h1>
+        <p class="text">I’m a software developer! I can help you build a product , 
+           feature or website Look through some of my work and experience!
+           If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <p class="connect">LET’S CONNECT</p>
+        <div class="social">
+            <a class="icon" href="#"><img src="./img/twitter.png" alt="twitter"></a>
+            <a class="icon" href="#"><img src="./img/Linkedin.png" alt="Linkedin"></a>
+            <a class="icon" href="#"><img src="./img/Mobile.png" alt="mobile"></a>
+            <a class="icon" href="#"><img src="./img/github.png" alt="github"></a>
+            <a class="icon" href="#"><img src="./img/succes.png" alt="vector"></a>
+
+        </div>
```

Comment:
- [ ] Well done but for the list of icons  you  should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
   * Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.